app/controllers/api/databases.php

```diff
@@ -1640,8 +1640,55 @@ function updateAttribute(
         Event $queueForEvents
```

Comment:
@abnegate Since these are only relation validations, maybe should be cleaner to leave it here, for example, the loop through the attributes we do not have in the createAttribute function 
What do you think